src/NumericalAlgorithms/Spectral/Projection.cpp

```diff
@@ -49,9 +49,10 @@ bool needs_projection(const Mesh<Dim>& mesh1, const Mesh<Dim>& mesh2,
                      });
 }
 
-const Matrix& projection_matrix_child_to_parent(
-    const Mesh<1>& child_mesh, const Mesh<1>& parent_mesh,
-    const ChildSize size) noexcept {
+const Matrix& projection_matrix_child_to_parent(const Mesh<1>& child_mesh,
+                                                const Mesh<1>& parent_mesh,
+                                                const ChildSize size,
+                                                const bool massive) noexcept {
```

Comment:
better name please...

Why not make this a separate function? It seems like the only thing shared are `ASSERT`s.

Also, since it's just the transpose, would it make more sense to handle this at the sight where the matrix is applied passing `'T'` to the `dgemm` call? I don't know since I have not used the code

Reply:
renamed in a fixup. I considered making this a separate function, but then the call site is more complicated. It's nice to be able to call this as `apply_matrices(projection_matrix(mesh, parent_mesh, operand_is_massive), operand, mesh)`.

Reply:
Sounds good :)